tests/src/dirs.proj

```diff
@@ -30,6 +30,8 @@
       <DisabledProjects Include="Performance\performance.csproj" />
       <DisabledProjects Include="Loader\classloader\generics\regressions\DD117522\Test.csproj" />
       <DisabledProjects Include="Loader\classloader\generics\GenericMethods\VSW491668.csproj" /> <!-- issue 5501 -->
+      <DisabledProjects Include="tracing\eventpipetrace\**" /> <!-- issue 15924 -->
+      <DisabledProjects Include="tracing\eventsource*\**" /> <!-- issue 15924 -->
```

Comment:
I would like to see this rebased so that this change isn't part of this PR

Reply:
Working on it